### PR TITLE
#6 Extra chapter's info in Readme-files

### PR DESCRIPTION
**Pull request by raman-m**
- Updated all README.md files for each chapter
- Updated main README.md for the book aka root folder
- Added cover page of the book to the main README.md file

### CODE REVIEW

File: trey-nash/12-threading-in-c#/README.md [hunk 1]
```diff
@@ -1,18 +1,114 @@
...
+the forefront because, rather than spending exorbitant amount of time and money to create faster
+processors, the industry has gravitated to creating processors with multiple cores. Therefore, I introduce
+the new Parallel Extensions and the Task Parallel Library (TPL) added to .NET 4.0.
+
+## Abstract
+The mere mention of multithreading can strike fear in the hearts of some programmers. For others, it
+fires them up for a good challenge. No matter how you react to the subject, multithreading is an area
+riddled with minefields. Unless you show due diligence, a threading bug can jump up and bite you—and
+bite you in a place where you cannot seem to find it easily. Threading bugs can be among the hardest to
+find and they are hard enough to find on a single-processor machine,; add more processors and cores,
```
Comment (fedotkin) [inline trey-nash/12-threading-in-c#/README.md:94]: please fix `,:`

Comment (raman-m) [reply trey-nash/12-threading-in-c#/README.md:94]: image
Well... This is copy-paste mistake from the original English book. I think the comma `,` should be removed.

Comment (raman-m) [reply trey-nash/12-threading-in-c#/README.md]: Fixed in [Update README.md for chapter 12. Fix comma](https://github.com/fedotkin/dotnet/pull/21/commits/7f4743ba2f74e2e6577e22cd3d2e503422cf217f)

**Review by fedotkin — APPROVED** [review]
very good job!
```diff
+ Approved  ✔️ 
```

**Pull request merged**